sdk/servicebus/service-bus/src/serializers/topicResourceSerializer.ts

```diff
@@ -283,6 +291,11 @@ export interface TopicProperties {
    * Specifies whether the topic should be partitioned
    */
   readonly enablePartitioning: boolean;
+
+  /**
+   * Specifies whether express entities are enabled on topic.
+   */
+  enableExpress: boolean;
```

Comment:
This is not a property that can be updated after a topic is created. We have been following the pattern of marking such properties as readonly (see enablePartitioning a few lines above)

```suggestion
  readonly enableExpress: boolean;
```